velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -40,6 +40,11 @@ class ContainerRowSerdeTest : public testing::Test,
       const TypePtr& type,
       vector_size_t numRows) {
     auto data = BaseVector::create(type, numRows, pool());
+    // Set all rows in data to NULL to verify that deserialize can clear nulls
+    // correctly.
+    for (auto i = 0; i < numRows; ++i) {
+      data->setNull(i, true);
```

Comment:
Can we set a group of consecutive nulls using mutableRawNulls?

Reply:
Existing APIs in Nulls.h and BitUtil.h doesn't support setting a range of nulls at once, so we would still need to loop over numRows on rawNulls even if we use mutableRawNulls.